src/test/java/org/gridsuite/modification/server/modifications/TwoWindingsTransformerCreationNodeBreakerTest.java

```diff
@@ -532,16 +532,16 @@ public void testCreateWithErrors() throws Exception {
         testNetworkModificationsCount(getGroupId(), 4);
     }
 
-    private void testCreateTwoWindingsTransformerInNodeBreaker(TwoWindingsTransformerCreationInfos twoWindingsTransformerCreationInfos, int actualSize) throws Exception {
+    private void testCreateTwoWindingsTransformerInNodeBreaker(TwoWindingsTransformerCreationInfos twoWindingsTransformerCreationInfos, int actualSize, int disconnector1, int disconnector2) throws Exception {
```

Comment:
We could get disconnector1 and disconnector2 dynamically. But not sure it's available in powsybl

Reply:
Indeed! I changed the test to avoid explicitly specifying the nodes in the test.